### ISSUE TITLE
Units

### ISSUE THREAD

**Issue by jrickmd** [bpennypacker/WeatherFlow-Indigo-Plugin#13]
Loving using 0.2. Thanks for your work!

Only thing I'm noticing that "might" be not working are the unit setting affecting the summary data. As an example, the "feels_like", "heat_index" variables both stay in C even when I have selected F as my unit of choice. Is this intentional to leave those alone?

Rick

**Comment by bpennypacker** [comment]
Hi Rick,

I actually made the decision to implement it this way, but maybe it doesn't make sense to. My only real concern with converting the raw summary data is how to best handle precision. For example, pressure defaults to mm of mercury. In converting that to inches of mercury it's essentially 1 mm == 0.03937008 inHg. The plugin is truncating the conversion to two decimal places, so using 0.04 in this case, but what if a user wants a different precision? I really don't want to overwhelm the plug-in with precision properties for everything if 90% of users will never use them and may just get confused by them.

-Bruce

**Comment by jrickmd** [comment]
Makes sense. I’ve already written python scripts to convert the things I want, just wanting to make sure that was your thought process.

Rick
